### DISCUSSION TITLE
Stellarium for M1

### DISCUSSION THREAD

**Discussion by JDftw** [Stellarium/stellarium#2059] [Q&A]
Hi! Would like to ask if Stellarium works on M1? Looking forward to it! Thank you! :)

**Comment by gzotti** [comment]
#1417

**Comment by alex-w** [comment]
<img width="1440" alt="Снимок экрана 2021-11-25 в 15 52 07" src="https://user-images.githubusercontent.com/88731/143409865-d16dd1f9-61ec-4b84-ad69-b618937f3602.png">

**Reply by bobuffs** [reply]
Just installed it my MacBook Pro M1 and it works superbly!

**Reply by mdolenga** [reply]
@bobuffs @alex-w  - Which version of Stellarium did you both install? I've tried with the latest, and a few earlier releases, and have the issue I reported below. Did you install anything else prior to running Stellarium upon which it might depend?

**Comment by mdolenga** [comment]
Not working for me. Installed without any complaints, but whenever I try to launch it, I get the "quit unexpectedly" error.

<img width="265" alt="Screen Shot 2021-12-05 at 2 18 22 AM" src="https://user-images.githubusercontent.com/10461666/144742501-9cc09e1c-f81c-4089-8499-97705b328c26.png">
<img width="422" alt="Screen Shot 2021-12-05 at 2 20 12 AM" src="https://user-images.githubusercontent.com/10461666/144742558-4916ab87-cb18-425f-bc97-1e303b11fa7d.png">

**Comment by bobuffs** [comment]
Hi Mdolenga,

 

The version I have installed is the latest (0.21.2).  I am running Monterey 12.0.1 on a MacBook Pro M1 2020. 8GB. All updates installed.

 

The only main items of software I otherwise have installed is SN8PP and AstroPlanner + a few other bits and pieces.  I also have Parallels  Windows emulator with Windows 11 that platforms other Win only astro programs.  All these were installed before Stellarium. Stellarium is installed on MACOS, not Parallels. I cannot think of anything that would influence the installation of Stellarium.

 

Let us know how you get on 😊

 

BR.

 

From: mdolenga ***@***.***>
Reply to: Stellarium/stellarium ***@***.***>
Date: Sunday, 5 December 2021 at 19:13
To: Stellarium/stellarium ***@***.***>
Cc: bobuffs ***@***.***>, Mention ***@***.***>
Subject: Re: [Stellarium/stellarium] Stellarium for M1 (Discussion #2059)

 

@bobuffs @alex-w - Which version of Stellarium did you both install? I've tried with the latest, and a few earlier releases, and have the issue I reported below. Did you install anything else prior to running Stellarium upon which it might depend?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by mdolenga** [reply]
Hi @bobuffs - I tried several versions, each of which failed. I then wondered whether being connected to an external monitor was problematic. So I disconnected it, rebooted the machine, and then reinstalled Stellarium. It works now. I then plugged the external monitor back in and it still works.

It is a new device and I've been setting things up on it to get it ready for use, perhaps there was some installation not quite complete which the reboot solved. I can't think of any other explanation.

Looks lovely on the M1 screen, and it's reporting ~95 fps consistently. I've never seen the app run this smooth before, wow.

**Comment by bobuffs** [comment]
Hi,

Interesting you mention the monitor connection. I didn’t think of that. I’ve had problems with Parallels running Win11 and an external monitor, and I thought the problem was with Parallels/Win11. 

I’m not wowed by Monterey as since upgrading I’ve experienced a number of issues that I hope will be resolved soon….

I agree Stellarium runs well and incredibly smooth, much nicer than any of the high cost apps….

Would just like an HR diagram!

Very best…
